conda-store-server/conda_store_server/server/auth.py

```diff
@@ -219,7 +216,7 @@ def compile_arn_regex(arn: str) -> re.Pattern:
           - "example-asdf"
           - "example-asdf/example-qwer"
         """
-        if not ARN_ALLOWED_REGEX.match(arn):
+        if not schema.ARN_ALLOWED_REGEX.match(arn):
```

Comment:
For consistency with `utils.py` and since it's better to check against what we expect vs what we do not
```suggestion
        if schema.ARN_ALLOWED_REGEX.match(arn) is None:
```